weblate_web/templates/features.html

```diff
@@ -67,7 +67,7 @@ <h3>{% trans "Integration API" %}</h3>
                 <a href="https://docs.weblate.org/en/latest/formats.html">
                     <div class="icon"><img src="{% static "img/icon-formats.svg" %}" /></div>
                     <h3>{% trans "Multiple File Formats" %}</h3>
-                    <p>{% trans "All widely used file formats for localization are supported, gettext, JSON, XLIFF or resource strings, etc." %}</p>
+                    <p>{% trans "Use any widely used file localization format, like gettext, JSON, XLIFF or resource strings, etc." %}</p>
```

Comment:
```suggestion
                    <p>{% trans "Use any common file localization format, like gettext, JSON, XLIFF or resource strings, etc." %}</p>
```
Avoids double "use" here.